bindings/python/README.md

```diff
@@ -1,10 +0,0 @@
-# Use udbserver in Python
-
-Check the [example](example.py) on how to use it.
-
-```sh
-$ sudo python3 setup.py install
-$ python3 example.py
-```
```

Comment:
I think we still need them (in `README.rst`).

Reply:
note that pypi doesn't support (and allow) most markdown syntax so even it must be convert-ed to plain text.